bosch_thermostat_client/db/nsc_icom_gateway.json

```diff
@@ -0,0 +1,292 @@
+{
+    "04.07.03": {
+        "heatingCircuits": {
+            "hastates": [
+                {
+                    "haname": "off",
+                    "boschname": [
+                        "off"
```

Comment:
you don't have `off` property according to `operation modes`. Delete this key.

Reply:
Just so we're on the same page, I guess you mean delete these rows?

```json
                {
                    "haname": "off",
                    "boschname": [
                        "off"
                    ]
                },
```